pulp_ansible/app/tasks/signature.py

```diff
@@ -22,6 +23,34 @@
 from pulpcore.plugin.models import SigningService, ProgressReport
 from pulpcore.plugin.sync import sync_to_async_iterable, sync_to_async
 from pulp_ansible.app.tasks.utils import get_file_obj_from_tarball
+from rest_framework import serializers
+
+
+def verify_signature_upload(data):
+    """The task code for verifying collection signature upload."""
```

Comment:
We would like to use the `ansible-galaxy colection verify` command, but I think it is [bugged](https://github.com/ansible/ansible/issues/77146), so this is a backup for right now.